source/hook.cpp

```diff
@@ -133,8 +133,13 @@ static int32_t HotKeyThread(void* arg) {
 			std::unique_lock<std::mutex> ulock(td->mtx);
 			for (auto& hk : td->hotkeys) {
 				bool allPressed = true;
-				for (key_t k : hk.second.keys) {
-					if (!isKeyDown(k))
+
+				for (std::pair<key_t, bool> k : hk.second.keys) {
+					bool isBound = k.second;
+					bool isPressed = isKeyDown(k.first);
+					
+					if ((isBound && !isPressed) ||
```

Comment:
Can be simplified into `isKeyDown(k.first) != k.second)`.

Reply:
Yeah I know, it's just easier to read.